pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -158,6 +158,10 @@
       "BaselineVersion": "5.0.0",
       "InboxOn": {}
     },
+    "Microsoft.NETCore.Platforms.Future": {
```

Comment:
You don't need to put any of the MS.Private package in here as no-one references them.